src/css-grids/cssgridsStyles.css

```diff
@@ -0,0 +1,92 @@
+body {
+  background-color: lavenderblush;
+  background-image: url(./backgroundimg.png);
```

Comment:
quotes around the url path is the expected way to write this
background-image: url("./backgroundimg.png");
also when naming things with two words do any of this: `background-img`, `backgroundImg` `background_img`.
just makes it easier to read! we code for humans.